Bugsnag/Assets/Bugsnag/Editor/SymbolUpload/BugsnagAndroidSymbolUploader.cs

```diff
@@ -0,0 +1,246 @@
+
+            var downloadProcess = new Process()
+            {
+                StartInfo = new ProcessStartInfo
+                {
+                    FileName = "curl",
+                    Arguments = $"-L {cliUrl} --output {downloadPath}",
+                    RedirectStandardOutput = true,
+                    RedirectStandardError = true,
+                    UseShellExecute = false,
+                    CreateNoWindow = true
+                }
+            };
+
+            downloadProcess.Start();
+            string downloadOutput = downloadProcess.StandardOutput.ReadToEnd();
+            string downloadError = downloadProcess.StandardError.ReadToEnd();
+            downloadProcess.WaitForExit();
+
+            if (downloadProcess.ExitCode != 0)
+            {
+                UnityEngine.Debug.LogError($"Failed to download the Bugsnag CLI tool. Error: {downloadError}");
+                return false;
+            }
+
+            UnityEngine.Debug.Log($"Bugsnag CLI tool downloaded successfully. Output: {downloadOutput}");
+            return true;
+        }
+
+        private bool MakeExecutable(string filePath)
+        {
+            if (Environment.OSVersion.Platform == PlatformID.Unix || Environment.OSVersion.Platform == PlatformID.MacOSX)
+            {
+                UnityEngine.Debug.Log($"Making the Bugsnag CLI executable with chmod on Unix/MacOS platform...");
+
+                var chmodProcess = new Process()
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = "chmod",
+                        Arguments = $"+x {filePath}",
+                        RedirectStandardOutput = true,
+                        RedirectStandardError = true,
+                        UseShellExecute = false,
+                        CreateNoWindow = true
+                    }
+                };
+
+                chmodProcess.Start();
+                string chmodOutput = chmodProcess.StandardOutput.ReadToEnd();
+                string chmodError = chmodProcess.StandardError.ReadToEnd();
+                chmodProcess.WaitForExit();
+
+                if (chmodProcess.ExitCode != 0)
+                {
+                    UnityEngine.Debug.LogError($"Failed to make the CLI executable. Error: {chmodError}");
+                    return false;
+                }
+                UnityEngine.Debug.Log($"Bugsnag CLI made executable successfully. Output: {chmodOutput}");
+            }
+
+            return true;
+        }
+
+        private bool RunBugsnagCliCommand(string cliPath, string arguments)
+        {
+            if (!File.Exists(cliPath))
+            {
+                UnityEngine.Debug.LogError($"Bugsnag CLI not found at {cliPath}");
+                return false;
+            }
+
+            UnityEngine.Debug.Log($"Running Bugsnag CLI tool with arguments: {arguments}");
```

Comment:
It'd be good to show the whole command to make it easy to copy-paste and debug:

```suggestion
            UnityEngine.Debug.Log($"Running BugSnag CLI:\n{cliPath} {arguments}");
```